### ISSUE TITLE
Calling ext-firestore-stripe-subscriptions-createCheckoutSession results in CORS error

### ISSUE THREAD

**Issue by wachauski** [stripe/stripe-firebase-extensions#99]
# Bug report

- Extension name: [e.g. `firestore-stripe-subscriptions`]

## Describe the bug

Added the Stripe extension for Firebase so that subscriptions can be called.  In order to make a payment without creating a new user that already exists in Stripe, I must create a session ID.  However, when I call the extension function, ext-firestore-stripe-subscriptions-createCheckoutSession, it shows a CORS error from both my localhost:4200 and https://dev.website.

Here's how I'm calling it:

async createSession() {
		const functionRef = firebase
		  .app()
		  .functions('us-central1')
		  .httpsCallable('ext-firestore-stripe-subscriptions-createCheckoutSession');
		const {data} = await functionRef();
		return data;
	}

Here's the error in the console:
Access to fetch at 'https://us-central1-....cloudfunctions.net/ext-firestore-stripe-subscriptions-createCheckoutSession' from origin 'http://localhost:4200' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

I had expected to get a session ID that I could then pass to stripe.redirectToCheckout so that it won't create a new user.

**Comment by thorsten-stripe** [comment]
Duplicate of https://github.com/stripe/stripe-firebase-extensions/issues/59. Please review the answers there.

**Comment by thorsten-stripe** [comment]
@wachauski sorry, I misread, `createCheckoutSession` is not a callable function and rather is triggered by a doc being inserted into the `checkout_session` collection: https://github.com/stripe/stripe-firebase-extensions/blob/next/firestore-stripe-subscriptions/POSTINSTALL.md#start-a-subscription-with-stripe-checkout

**Comment by wachauski** [comment]
Thanks, Thorsten.  That's a useful link.  By the looks of the function, it needs to be created on the server-side.  Right?  If so, won't I run into the same CORS issue when I try to call it form the client?  I don't see how I can access Firestore otherwise.

**Comment by thorsten-stripe** [comment]
@wachauski all your interactions with the extension are client-side. I'd recommend to read the docs. They can be found in the extension settings in your Firebase console: https://console.firebase.google.com/u/0/project/_/extensions/instances/firestore-stripe-subscriptions
![image](https://user-images.githubusercontent.com/23213994/99138290-9bf75a80-266a-11eb-9030-0d9dec6793c8.png)

The docs also link an example integration that you might find helpful: https://github.com/stripe-samples/firebase-subscription-payments
